timepiece/crm/models.py

```diff
@@ -315,8 +317,10 @@ class PaidTimeOffRequest(models.Model):
 
     class Meta:
         ordering = ('user_profile', '-pto_start_date',)
-        permissions = (("can_approve_pto_requests", "Can approve PTO requests"),
-                       ("can_process_pto_requests", "Can payroll process PTO requests"), )
+        permissions = (
+            ("can_approve_pto_requests", "Can approve PTO requests"),
```

Comment:
Can you remove the `can_` here and make it singular?  I know this was from me originally, but it should just be `approve_pto_request` to follow standard practice.  Can you do that throughout this PR, where I may have messed it up in the past?
We will want to do a `git grep -n can_approve_pto_requests` after making the change to see if there are other places in code we reference that permission.